packages/app/src/KubernetesDashboardPage.tsx

```diff
@@ -472,30 +479,37 @@ const NodesTable = ({
               ) : (
                 <tbody>
                   {nodesList.map(node => (
-                    <tr key={node.name}>
-                      <td>{node.name || 'N/A'}</td>
-                      <td>
-                        {node.ready === 1 ? (
-                          <Badge color="green" fw="normal" tt="none" size="md">
-                            Ready
-                          </Badge>
-                        ) : (
-                          <Badge color="red" fw="normal" tt="none" size="md">
-                            Not Ready
-                          </Badge>
-                        )}
-                      </td>
-                      <td>
-                        {formatNumber(
-                          node.cpuAvg,
-                          K8S_CPU_PERCENTAGE_NUMBER_FORMAT,
-                        )}
-                      </td>
-                      <td>
-                        {formatNumber(node.memAvg, K8S_MEM_NUMBER_FORMAT)}
-                      </td>
-                      <td>{node.uptime ? formatUptime(node.uptime) : '–'}</td>
-                    </tr>
+                    <Link key={node.name} href={getLink(node.name)}>
```

Comment:
any chance we can add anchor tags to these so they're more clicky?

Reply:
not very trivial, let's descope for now :D